data/json/itemgroups/Clothing_Gear/clothing.json

```diff
@@ -3217,84 +3217,93 @@
-      [ "helmet_conical", 30 ],
-      [ "armor_lamellar", 20 ],
-      [ "armor_lorica", 25 ],
-      [ "armor_samurai", 50 ],
-      [ "helmet_kabuto", 50 ],
-      [ "helmet_larmor", 40 ],
-      [ "helmet_nasal", 50 ],
-      [ "helmet_galea", 40 ],
-      [ "boots_larmor", 40 ],
-      [ "armor_larmor", 40 ],
-      [ "armguard_larmor", 40 ],
-      [ "vambrace_larmor", 20 ],
-      [ "gambeson", 50 ],
-      [ "gambeson_pants", 40 ],
-      [ "legguard_metal", 10 ],
-      [ "helmet_corinthian", 45 ],
-      [ "armor_cuirass", 25 ],
-      [ "legguard_bronze", 20 ]
+      { "item": "armor_junk_plate", "prob": 60, "damage": [ 4, 4 ] },
+      { "item": "gloves_plate", "prob": 60, "damage": [ 4, 4 ] },
+      { "item": "boots_plate", "prob": 60, "damage": [ 4, 4 ] },
+      { "item": "armor_junk_lightplate", "prob": 45, "damage": [ 4, 4 ] },
+      { "item": "cuirass_lightplate", "prob": 45, "damage": [ 4, 4 ] },
+      { "item": "armguard_lightplate", "prob": 30, "damage": [ 4, 4 ] },
+      { "item": "legguard_lightplate", "prob": 30, "damage": [ 4, 4 ] },
+      { "item": "helmet_barbute", "prob": 50, "damage": [ 4, 4 ] },
+      { "item": "helmet_conical", "prob": 30, "damage": [ 4, 4 ] },
+      { "item": "armor_lamellar", "prob": 20, "damage": [ 4, 4 ] },
+      { "item": "armor_lorica", "prob": 25, "damage": [ 4, 4 ] },
+      { "item": "armor_samurai", "prob": 50, "damage": [ 4, 4 ] },
+      { "item": "helmet_kabuto", "prob": 50, "damage": [ 4, 4 ] },
+      { "item": "helmet_larmor", "prob": 40, "damage": [ 4, 4 ] },
+      { "item": "helmet_nasal", "prob": 50, "damage": [ 4, 4 ] },
+      { "item": "helmet_galea", "prob": 40, "damage": [ 4, 4 ] },
+      { "item": "boots_larmor", "prob": 40, "damage": [ 4, 4 ] },
+      { "item": "armor_larmor", "prob": 40, "damage": [ 4, 4 ] },
+      { "item": "armguard_larmor", "prob": 40, "damage": [ 4, 4 ] },
+      { "item": "vambrace_larmor", "prob": 20, "damage": [ 4, 4 ] },
+      { "item": "gambeson", "prob": 50, "damage": [ 4, 4 ] },
+      { "item": "gambeson_pants", "prob": 40, "damage": [ 4, 4 ] },
+      { "item": "legguard_metal", "prob": 10, "damage": [ 4, 4 ] },
+      { "item": "helmet_corinthian", "prob": 45, "damage": [ 4, 4 ] },
+      { "item": "armor_cuirass", "prob": 25, "damage": [ 4, 4 ] },
+      { "item": "legguard_bronze", "prob": 20, "damage": [ 4, 4 ] }
     ]
   },
   {
     "type": "item_group",
     "id": "museum_armor_torso",
     "items": [
-      [ "armor_junk_plate", 60 ],
-      [ "armor_junk_lightplate", 45 ],
-      [ "cuirass_lightplate", 45 ],
-      [ "armor_lamellar", 20 ],
-      [ "armor_lorica", 25 ],
-      [ "armor_samurai", 50 ],
-      [ "armor_larmor", 40 ],
-      [ "gambeson", 50 ],
-      [ "armor_cuirass", 25 ],
-      [ "chainmail_junk_suit", 10 ],
-      [ "chainmail_junk_vest", 3 ]
+      { "item": "armor_junk_plate", "prob": 60, "damage": [ 4, 4 ] },
+      { "item": "armor_junk_lightplate", "prob": 45, "damage": [ 4, 4 ] },
```

Comment:
junk plate already has damage reflected in it's base stats.  

Also variety is probably best.   Cuts down on the pinata aspect but leaves a location still as a location of interest.

Reply:
Do you think i need to make junk plate to be in perfect condition?